### PR TITLE
Clarify `/ready` endpoint description

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -45,11 +45,15 @@ plain text. The text format is following the exporter format used by `Prometheus
 /ready
 ~~~~~~
 
-This endpoint will return a HTTP status code only, no content is returned.
-The status code indicates if the CrateDB node is able to process SQL
-statements. This could not be the case e.g. when a node is gracefully shutdown.
-The readiness endpoint can be usefull for load blancers to know when to remove a
-node from the pool.
+This endpoint will return a HTTP status code only, without content. The status
+code indicates if the *individual* node is ready to process statements.
```
Comment (chaudum) [inline README.rst]: if -> whether

File: README.rst [hunk 2]
```diff
@@ -45,11 +45,15 @@ plain text. The text format is following the exporter format used by `Prometheus
...
+
+For a node to be able to process statements the cluster state also has to be
+okay. But if that is the case is *not* reflected by this endpoint.
```
Comment (chaudum) [inline README.rst]: Not sure what you mean with the second sentence.

**Comment by mfussenegger** [comment]
@chaudum pushed a fixup, re-phrased it again slightly

File: README.rst [hunk 3]
```diff
@@ -45,11 +45,15 @@ plain text. The text format is following the exporter format used by `Prometheus
 /ready
 ~~~~~~
 
-This endpoint will return a HTTP status code only, no content is returned.
-The status code indicates if the CrateDB node is able to process SQL
-statements. This could not be the case e.g. when a node is gracefully shutdown.
-The readiness endpoint can be usefull for load blancers to know when to remove a
-node from the pool.
+This endpoint will return a HTTP status. The status code indicates whether he
```
Comment (andreidan) [inline README.rst]: "whether the"

otherwise, I think it makes more sense now

**Pull request merged**